### PR TITLE
[pt] Translate distillery lesson

### PR DESCRIPTION
**Pull request by gugutz**
Add portuguese translation for the Distillery lesson.

### CODE REVIEW

File: pt/lessons/libraries/distillery.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
+---
+version: 2.0.1
+title: Distillery (Básico)
+---
+
+Distillery é um gerenciador de releases escrito em Elixir puro. Ele permite que você produza releases que podem ser instaladas em outros lugares com pouca ou nenhuma configuração.
```
Comment (thiamsantos) [inline pt/lessons/libraries/distillery.md]:
```suggestion
Distillery é um gerenciador de releases escrito em Elixir puro. Ele permite que você produza releases que podem ser rodadas em outros lugares com pouca ou nenhuma configuração.
```
I don't know if `instaladas` is the best translation for `deployed` is this specific case. In my opinion, here `instaladas` doesn't fully translate the meaning of the word. @elixirschool/portuguese Do you any suggestion on how we can better translate this?

File: pt/lessons/libraries/distillery.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+
+> Quando você escrever uma ou mais aplicações, você talvez queira criar um sistema completo com estas aplicações e um subconjunto das aplicações Erlang/OTP. Isto é chamado de release. - [Documentação do Erlang](http://erlang.org/doc/design_principles/release_structure.html)
+
+> Releases permitem deployment simplificados: elas são auto-contidas, e provêm tudo o que for necessário para iniciar a release; elas são facilmente administradas via o shell script provido para abrir um console remoto, iniciar/parar/reiniciar a release, iniciar no background, enviar comandos remotos, e mais. Além disso, elas são artefatos arquiváveis, o que significa que você pode restaurar uma release antiga do seu tarball em qualquer momento no futuro (salvo incompatibilidades com o SO ou bibliotecas do sistema). O uso de releases também é um pré-requisito para realizar hot upgrades e downgrades, um dos recursos mais poderosos da VM do Erlang. - [Documentação do Distillery](https://hexdocs.pm/distillery/introduction/understanding_releases.html)
+
+Uma release irá conter o seguinte:
+* uma pasta /bin
+  * Isto contém um script que é o ponto de início para rodar a sua aplicação inteira.
```
Comment (thiamsantos) [inline pt/lessons/libraries/distillery.md]:
```suggestion
  * Esta contém um script que é o ponto de início para rodar a sua aplicação inteira.
```
:nail_care: In my opinion reads more naturally.

File: pt/lessons/libraries/distillery.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+* uma pasta /releases
+  * Isto contém metadados sobre a release assim como também hooks e comandos customizados.
+* Um /erts-VERSION
+  * Isto contém o runtime do Erlang que irá permitir que uma máquina execute a sua aplicação sem necessitar ter o Erlang ou Elixir instalados.
+
+
+### Iniciando/instalação
+
+Para adicionar o Distillery no seu projeto, adicione-o como uma dependência no seu arquivo `mix.exs`. *Nota* - se você estiver trabalhando numa aplicação umbrella isto deve estar no mix.exs na raiz do seu projeto
+
```
Comment (thiamsantos) [inline pt/lessons/libraries/distillery.md]: :scissors:

File: pt/lessons/libraries/distillery.md (hunk 1 above)
Comment (gugutz) [reply pt/lessons/libraries/distillery.md]: Yeah i agree. I also thought about using `distribuidas` but that doens't seem to to fit well either.

IMO i would maintain the original word as well. The issue as i see it are the conjugations, like `deployar`, `deployadas`, and so on. But those still sounds better than the other translations.

Just let me know and ill change to whatever you guys think its best. :)

File: pt/lessons/libraries/distillery.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+Nós fizemos algumas coisas aqui:
+- `server` - inicia o endpoint HTTP da aplicação Cowboy no início do aplicação
+- `root` - define a raiz da aplicação que é onde os arquivos estáticos são servidos
+- `version` - quebra o cache da aplicação quando a versão da mesma sofre um hot upgrade
+- `port` - alterar a porta para ser setada por uma variável de ambiente permite que passamos o número da porta quando estivermos iniciando a aplicação. Quando iniciamos a aplicação, podemos suprir a porta executando `PORT=4001 _build/prod/rel/book_app/bin/book_app foreground`
+
+Se você executou o comando acima, você talvez tenha notado que a sua aplicação crashou porquê é incapaz de conectar ao banco de dados já que nenhum banco de dados atualmente existe. Isto pode ser retificado executando um comando `mix` do Ecto. No seu terminal, digite o seguinte:
+
+
```
Comment (thiamsantos) [inline pt/lessons/libraries/distillery.md]: :scissors:

File: pt/lessons/libraries/distillery.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+* pre_start
+* post_start
+* pre/post_configure
+* pre/post_stop
+* pre/post_upgrade
+
+
+Para o nosso propósito, iremos estar utilizando o hook `post_start` para executar as migrações da nossa aplicação em produção. Primeiro vamos criar uma nova tarefa da release chamada `migrate`. Uma tarefa de release é um módulo que podemos chamar pelo terminal e que contém código que é separado do funcionamento interno da nossa aplicação. Isto é útil para tarefas que a aplicação em si não precisa tipicamente executar.
+
+
```
Comment (thiamsantos) [inline pt/lessons/libraries/distillery.md]: :scissors:

File: pt/lessons/libraries/distillery.md [hunk 6]
```diff
@@ -0,0 +1,243 @@
+---
+version: 3.0.1
```
Comment (thiamsantos) [inline pt/lessons/libraries/distillery.md]:
```suggestion
version: 2.0.1
```
We should use the same version of the english text.

File: pt/lessons/libraries/distillery.md (hunk 1 above)
Comment (anderkonzen) [reply pt/lessons/libraries/distillery.md]: I would keep the original word in this case, `deployed`, as Thiago pointed out.

Comment (gugutz) [reply pt/lessons/libraries/distillery.md]: @anderkonzen just to be sure, the current phrase is like this:
`... que podem ser **rodadas** em outros lugares...`

do i change it to:
`... que podem ser **deployed** em outros lugares...`

or : 
`... que podem ser **deployadas** em outros lugares...`
?

Comment (anderkonzen) [reply pt/lessons/libraries/distillery.md]: I think `... que podem ser deployed em outros lugares ...`. Still looks a bit weird to me but seems correct at the moment.

I also saw this word is used in the "Custom commands" section, we can adjust there as well.

Comment (gugutz) [reply pt/lessons/libraries/distillery.md]: Done. :)
Let me know if anything else comes up.

Comment (thiamsantos) [reply pt/lessons/libraries/distillery.md]: Nice :)

**Comment by thiamsantos** [comment]
Looks good to me. @anderkonzen do you have anything else to add? Or we are good to go?

**Comment by anderkonzen** [comment]
Everything looks good to me! Thanks @gugutz !

**Pull request merged**